spritz/app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,21 @@
+<resources>
+
+    <!-- Base application theme. -->
```

Comment:
Could remove these comments

Reply:
Done in 7310506c771437dedd2ac58ddc074a5bf6c6a982.